Gemfile

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 # BEFORE changing this file, read https://wiki.doximity.com/articles/gemfile-maintenance
-source "https://artifacts.dox.support/repository/gems"
+# source "https://artifacts.dox.support/repository/gems"
+source "https://rubygems.org"
```

Comment:
Required for circleci because we are using a public image that doesn't have credentials.

Reply:
We may as well remove the commented reference